addons/escoria-ui-simplemouse/game.gd

```diff
@@ -239,8 +244,8 @@ func left_click_on_inventory_item(inventory_item_global_id: String, event: Input
 		escoria.action_manager.ACTION.ITEM_LEFT_CLICK,
 		[inventory_item_global_id, event]
 	)
-
-	if escoria.action_manager.current_action == "use":
+	
```

Comment:
doc8 will complain about the tab here :D

Reply:
I'm not sure if doc8 runs on the non-documentation repos, but maybe I'm wrong. Either way, will remove!